src/ceph-disk

```diff
@@ -1719,6 +1804,9 @@ def main_prepare(args):
 
         if args.zap_disk is not None:
             zap(args.data)
```

Comment:
Great ! Could you make a separate pull request with this so it can be used to resolve http://tracker.ceph.com/issues/13291 ?